sdcm/nemesis.py

```diff
@@ -4318,6 +4319,119 @@ def disrupt_add_remove_mv(self):
         finally:
             self.unset_current_running_nemesis(cql_query_executor_node)
 
+    def disrupt_bootstrap_streaming_error(self):
```

Comment:
Add some docs - most of disruptions have.

Reply:
Done